JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -115,16 +115,16 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
     if(options[["distribution"]] == "benford"){
 
       procedureText <- base::switch(options[["digits"]],
-                                    "first" = gettextf("Benford's law states that in many naturally occurring collections of numbers, the leading significant number is likely to be small. The goal of this procedure is to determine to which extent the first leading numbers in the population follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that do not conform to Benford's law might need further verification.", confidenceLabel),
-                                    "firstSecond" = gettextf("Benford's law states that in many naturally occurring collections of numbers, the leading significant number is likely to be small. The goal of this procedure is to determine to which extent the first two leading numbers in the population follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that do not conform to Benford's law might need further verification.", confidenceLabel),
-                                    "last" = gettextf("Benford's law states that in many naturally occurring collections of numbers, the leading significant number is likely to be small. The goal of this procedure is to determine to which extent the last numbers in the population follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that do not conform to Benford's law might need further verification.", confidenceLabel))
+                                    "first" = gettextf("Benford's law states that in many naturally occurring collections of numerical observations, the leading significant digit is likely to be small. The goal of this procedure is to determine to which extent the first leading digits in the data set follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that do not conform to Benford's law might need further verification.", confidenceLabel),
```

Comment:
"Benford's law states that in many naturally occurring collections of numerical observations, the leading significant digit is likely to be small. The goal of this procedure is to determine to **what** extent the first leading digits in the data set follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that do not conform to Benford's law might need further **investigation**."

I'm not so sure about the last sentence. I changed it to further investigation rather than verification, because I don't know what to verify.

Reply:
Sometimes Benford's law is used to detect fraud in financial data. Then the thing to verify should be whether the numbers are correct. But I will change which into what.

Reply:
Verification that it is correct? Verification that there's no fraud? You cannot verify that, unless you study all the entries. I think that you can however recommend further investigation, rather than verification.